Neos.EventSourcedContentRepository/Classes/Command/NodeMigrationCommandController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+namespace Neos\EventSourcedContentRepository\Command;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\ContentRepository\Domain\ContentStream\ContentStreamIdentifier;
+use Neos\ContentRepository\Migration\Domain\Factory\MigrationFactory;
+use Neos\EventSourcedContentRepository\Domain\Context\Migration\NodeMigrationService;
+use Neos\EventSourcedContentRepository\Domain\ValueObject\WorkspaceName;
+use Neos\Flow\Cli\CommandController;
+use Neos\ContentRepository\Migration\Exception\MigrationException;
+use Neos\ContentRepository\Migration\Domain\Model\MigrationConfiguration;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * Command controller for tasks related to node migration.
+ *
+ * @Flow\Scope("singleton")
+ */
+class NodeMigrationCommandController extends CommandController
```

Comment:
This turns the command into `nodemigration:migrate`. I assume that you did that on purpose to prevent naming clashes with `node:migrate`, right?